src/Adapter/SpecificPrice/CommandHandler/AddSpecificPriceHandler.php

```diff
@@ -36,8 +39,17 @@
 use PrestaShopException;
 use SpecificPrice;
 
+@trigger_error(
+    sprintf(
+        '%s is deprecated since version 1.7.8.0 and will be removed in the next major version.',
```

Comment:
```suggestion
        '%s is deprecated since version 1.7.9.0 and will be removed in the next major version.',
```

Reply:
@jolelievre did deprecate this handler related command in 1.7.8 but forgot to add the needed annotation in handler (and maybe interface and some other classes related to AddSpecificPriceCommand. 
So now it seems that the command is deprecated in 178 but the handler would be deprecated in 179 even though they cannot be used separately?
idk how to act in this situation.

Reply:
If it wasn't mark as deprecated in the previous version, it can't be considered as deprecated, because people using 1.7.8.x will not receive a deprecation notice.

Reply:
Ok, my bad